integration-tests/upgradeable-contracts/delegator/delegatee2/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "delegatee2"
```

Comment:
Can you add a mention of the lock/unlock mechanism and the `delegate2` package to https://github.com/paritytech/ink/tree/430a831b1e4084e154744910f36a9f40516699fe/integration-tests/upgradeable-contracts#delegator?

Reply:
Done :+1:

Reply:
Ah oops I did that in the wrong PR but it is here: https://github.com/paritytech/ink/pull/2123/files#diff-7cf9db86b007053cf1ad83c5196420d3e5c582a7b3dbbc5d73edca22b4a452fa